cetmix_tower_server/models/cx_tower_server_template.py

```diff
@@ -251,7 +259,7 @@ def _get_fields_tower_server(self):
             "server_log_ids",
         ]
 
-    def _prepare_server_values(self, **kwargs):
+    def _prepare_server_values(self, pick_all_template_variables=True, **kwargs):
```

Comment:
with these changes, `prepare_server_values` is not a bit long, should we refactor to make it shorter and easier to understand?